src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
My thinking on the `gapic/` tag is that it represents the version of the GAPIC client library, not necessarily the version of the GAPIC generator itself.

That encapsulates the scenario that @geigerj described. We don't track the version of an API beyond its major, but we could use the same version of the code generator several times as the API is enhanced within a given major. By tracking a strictly monotonically increasing version number for the generated GAPIC library, we'll have metrics that differentiate between those releases.